guides/quickstart.md

```diff
@@ -0,0 +1,175 @@
+# Quickstart: How to configure and run the analyzer
+
+The code for this example is available in the `go-flow-levee/guides/quickstart` directory.
+
+## Context
+
+Suppose you have the following piece of code:
+
+`auth.go`
+```go
+package auth
```

Comment:
Nit: For filenames in Markdown, I feel like I've typically seen it as a leading comment, i.e.

```golang
// auth.go
package auth
// ...
```

The spacing staggers a bit otherwise.